lua/gatekeeper/staticlib.lua

```diff
@@ -19,6 +19,15 @@ function get_numa_table (net_conf)
 	return numa_table
 end
 
+function count_numa_nodes (numa_table)
+	local count = 0
+	for _ in pairs(numa_table) do
+		count = count + 1
+	end
+    -- Skip the field __net_conf@numa_table.
```

Comment:
Drop "@numa_table" since the field is just `__net_conf`. `numa_table` is a field of `struct net_config`.

Reply:
A better solution here is to use `all_ipairs()` instead of `pairs()`.